assets/frontend/invoices/services.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+  'use strict'
+
+  angular
+    .module('invoices.portal')
+    .service('InvoicesServices', InvoicesServices)
+  ;
+  /* INVOICE SERVICE: this wil fetch data from the backend
```

Comment:
wrong spelling. Services doesn't only get data from the backend.

*FYI: services should contain all the logic of the app